spark/src/main/kotlin/com/adevinta/spark/tools/modifiers/Layout.kt

```diff
@@ -0,0 +1,132 @@
+import androidx.compose.ui.graphics.PathEffect
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.layout.Measurable
+import androidx.compose.ui.layout.MeasureResult
+import androidx.compose.ui.layout.MeasureScope
+import androidx.compose.ui.layout.boundsInWindow
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.node.LayoutModifierNode
+import androidx.compose.ui.node.modifierElementOf
+import androidx.compose.ui.platform.LocalAutofill
+import androidx.compose.ui.platform.LocalAutofillTree
+import androidx.compose.ui.unit.Constraints
+import androidx.compose.ui.unit.Dp
+
+/**
+ * Ignore the parent padding by [horizontalPadding].
+ * Be careful since concretely it makes the maxWidth bigger which could make this composable clipped by the parent
+ * width if misused
+ * @param horizontalPadding The horizontal padding to ignore from the parent
+ */
+@OptIn(ExperimentalComposeUiApi::class)
+fun Modifier.ignoreParentHorizontalPadding(horizontalPadding: Dp) = this then modifierElementOf(
+    params = horizontalPadding,
+    create = { IgnoreParentHorizontalPaddingModifier(horizontalPadding) },
+    update = { it.horizontalPadding = horizontalPadding },
+    definitions = {
+        name = "ignoredParentPadding"
+        properties["horizontalPadding"] = horizontalPadding
+    },
+)
+
+@OptIn(ExperimentalComposeUiApi::class)
+private class IgnoreParentHorizontalPaddingModifier(var horizontalPadding: Dp) : Modifier.Node(), LayoutModifierNode {
+    override fun MeasureScope.measure(
+        measurable: Measurable,
+        constraints: Constraints,
+    ): MeasureResult {
+
+        val placeable = measurable.measure(
+            constraints.copy(
+                maxWidth = constraints.maxWidth + (horizontalPadding * 2).roundToPx(),
+            ),
+        )
+        return layout(placeable.width, placeable.height) {
+            placeable.place(x = 0, y = 0)
+        }
+    }
+}
+
+/**
+ * Add a dashed border on a Composable
+ */
+fun Modifier.dashedBorder(width: Dp, radius: Dp, color: Color) = drawBehind {
+    drawIntoCanvas {
+        val paint = Paint()
+            .apply {
+                strokeWidth = width.toPx()
+                this.color = color
+                style = PaintingStyle.Stroke
+                pathEffect = PathEffect.dashPathEffect(floatArrayOf(10f, 10f), 0f)
+            }
+        it.drawRoundRect(
+            left = width.toPx(),
+            top = width.toPx(),
+            right = size.width - width.toPx(),
+            bottom = size.height - width.toPx(),
+            radiusX = radius.toPx(),
+            radiusY = radius.toPx(),
+            paint = paint,
+        )
+    }
+}
+
+/**
+ * Add autofill support to a given Composable
+ *
+ * Taken from [Autofill with Jetpack Compose · Bryan Herbst](https://bryanherbst.com/2021/04/13/compose-autofill/)
+ */
+// TODO-scott.rayapoulle.ext (03-51-2023): Replace by official implementation on b/176949051
```

Comment:
nit: Use GitHub alias

Reply:
done in 2b254de8

This one: @soulcramer ?